### PR TITLE
pre-workshop survey questions

### PR DESCRIPTION
**Pull request by psteinb**
This file lists questions that can be included in a pre-workshop survey in order for instructors to get to know their learners **BEFORE** as workshop. I'd be curious for reviews.

Note: At best, this document should be referenced in some kind of `instructors guide`. But if I am not mistaken, `_extras/guide.md` makes an attempt to be such an instructors guide.

### CODE REVIEW

File: _extras/learner-prerequisites.md [hunk 1]
```diff
@@ -0,0 +1,136 @@
+---
+title: Learner Prerequisites
+---
+
+This page collects questions that learners can answer in order to judge if they 
+bring all prequisites to the course.
```
Comment (tkphd) [inline _extras/learner-prerequisites.md]: prerequisites

**Review by annajiat — COMMENTED** [review]
added Python Setup 1, 2

**Comment by annajiat** [comment]
Regarding the multiple choice options, 
Proposed: 1. I can do that. Give me a python environment and I'll show you.
Current: 1. I can do that. Give me something that understands python and I'll show you.

File: _extras/learner-prerequisites.md (hunk 1 above)
Comment (psteinb) [reply _extras/learner-prerequisites.md]: fixed in 
c9f0a24

**Comment by psteinb** [comment]
@annajiat I think we should choose, what is more appropriate.

**Comment by tkphd** [comment]
Rebased against gh-pages to resolve build errors, then force-pushed to keep a tidy history.

**Review by tkphd — APPROVED** [review]
LGTM!

**Pull request merged**